website/pages/guides/[[...page]].jsx

```diff
@@ -10,15 +10,12 @@ import {
 const BASE_ROUTE = 'guides'
 const NAV_DATA = 'data/guides-nav-data.json'
 const CONTENT_DIR = 'content/guides'
+const MAIN_BRANCH = 'master'
+const PRODUCT = { name: productName, slug: productSlug }
 
-export default function GuidesLayout(props) {
+export default function IntroLayout(props) {
   return (
-    <DocsPage
-      baseRoute={BASE_ROUTE}
-      mainBranch="master" // used for "edit on this page", default "main"
-      product={{ name: productName, slug: productSlug }}
-      staticProps={props}
-    />
+    <DocsPage baseRoute={BASE_ROUTE} product={PRODUCT} staticProps={props} />
```

Comment:
The API for `docs-page` has changed slightly since the pre-release. Main change is how the `Edit this page` URL is constructed (now happens in `getStaticProps` rather than within the component).